presto-native-execution/presto_cpp/main/http/HttpServer.cpp

```diff
@@ -27,12 +27,22 @@ void sendOkResponse(proxygen::ResponseHandler* downstream) {
 }
 
 void sendOkResponse(proxygen::ResponseHandler* downstream, const json& body) {
-  proxygen::ResponseBuilder(downstream)
-      .status(http::kHttpOk, "OK")
-      .header(
-          proxygen::HTTP_HEADER_CONTENT_TYPE, http::kMimeTypeApplicationJson)
-      .body(body.dump())
-      .sendWithEOM();
+  // nlohmann::json throws when it finds invalid UTF-8 characters. In that case
```

Comment:
@spershin @mbasmanova Do we know how invalid characters are coming in the body? Does the replacement fail the query and keep the server alive?